tji-data.css

```diff
@@ -68,7 +68,20 @@ $bodyFont: 'Raleway', sans-serif; // 400, 600 weight
 }
 */
 
```

Comment:
since we're trying to clean up our random css files and stick to sass files, could you stick your style definition for the loader in the sass folder. Maybe create a _helpers.scss in sass/tji/ and add that to _tji.scss and then rebuild.

Reply:
done